buildroot-external/patches/occu/0154-WebUI-DynamicResizing.patch

```diff
@@ -113,6 +113,138 @@
        gotoLoginPage = function() {
          location.href = "/login.htm";
        };
+--- occu/WebUI/www/rega/pages/tabs/control/devices.htm.orig
++++ occu/WebUI/www/rega/pages/tabs/control/devices.htm
+@@ -13,14 +13,9 @@
+   arItems = new Array();
+ </script>
+ 
+-<table width="100%" class="CLASS03900" cellpadding="0" cellspacing="0">
+-  <colgroup>
+-    <col style="width:20%;"/>
+-    <col style="width:80%;"/>
+-  </colgroup>
+-  <tr>
++<div id="devicepage" class="CLASS03900" style="width:100%">
+     <!-- Linke Spalte -->
+-    <td id="contentLeft" class="hidden">
++    <div id="contentLeft" class="hidden" style="width:20%;float:left;overflow:auto;">
+       <table class="filterTable CLASS03901 j_translate" cellpadding="0" cellspacing="0">
+         <tr>
+           <td style="vertical-align: top; border: 0px">
+@@ -120,10 +115,10 @@
+           </td>
+        </tr> 
+      </table>
+-   </td>
++   </div>
+ 
+     <!-- Rechte Spalte -->
+-    <td id="contentRight">
++    <div id="contentRight">

```

Comment:
Please note that the css definition for `contentRight` is:

```css
#contentRight
{   
  height: 100%;                         
  border-left: solid 5px $_(navBarSeparator); /* #ffffff */ /* Trennlinie zwischen Navigation und Hauptber
  /* padding-top: 10px; */
  vertical-align: top;
}
```

However, with the new `<div>` way the `border-left` statement does not work anymore. However, would be great to have that grey separator bar between contentLeft and contentRight back. So please revise.

Reply:
I think, `#contentRight` needs `overflow: auto`